assembler/templates/archive.html

```diff
@@ -1,40 +1,49 @@
-    <div class="wp-block-group" style="border-radius:4px"><!-- wp:group {"style":{"border":{"radius":"4px","top":{"style":"none","width":"0px"},"right":{"style":"none","width":"0px"},"bottom":{"style":"none","width":"0px"},"left":{"style":"none","width":"0px"}}},"backgroundColor":"tertiary","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"center"}} -->
-    <div class="wp-block-group has-tertiary-background-color has-background" style="border-radius:4px;border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-style:none;border-left-width:0px"><!-- wp:post-terms {"term":"category","textAlign":"center","separator":"  ","style":{"layout":{"selfStretch":"fill"},"spacing":{"padding":{"top":"6px","right":"6px","bottom":"6px","left":"6px"}}}} /--></div>
-    <!-- /wp:group --></div>
-    <!-- /wp:group -->
-    
-    <!-- wp:post-featured-image {"isLink":true,"aspectRatio":"3/2"} /-->
-    
-    <!-- wp:post-title {"isLink":true,"fontSize":"x-large"} /-->
-    
-    <!-- wp:post-excerpt {"showMoreOnNewLine":false,"excerptLength":50} /-->
-    
-    <!-- wp:post-date /-->
-    
-    <!-- wp:spacer {"height":"40px"} -->
-    <div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-    <!-- /wp:spacer --></div>
-    <!-- /wp:group -->
-    <!-- /wp:post-template -->
-    
-    <!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
-    <div class="wp-block-group alignwide"><!-- wp:query-pagination {"align":"wide","layout":{"type":"flex","justifyContent":"space-between"}} -->
-    <!-- wp:query-pagination-previous /-->
-    
-    <!-- wp:query-pagination-next /-->
-    <!-- /wp:query-pagination --></div>
-    <!-- /wp:group --></main>
-    <!-- /wp:query --></div>
-    <!-- /wp:group -->
-
-<!-- wp:template-part {"slug":"footer"} /-->
\ No newline at end of file
+<!-- wp:group {"tagName":"main","metadata":{"name":"Main"},"align":"full","layout":{"type":"constrained"}} -->
+<main class="wp-block-group alignfull"><!-- wp:spacer {"height":"var:preset|spacing|20"} -->
+<div style="height:var(--wp--preset--spacing--20)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:query-title {"type":"archive","textAlign":"center","showPrefix":false,"align":"wide","style":{"typography":{"lineHeight":"1"}},"fontSize":"large"} /-->
+
+<!-- wp:spacer {"height":"var:preset|spacing|10"} -->
+<div style="height:var(--wp--preset--spacing--10)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:query {"queryId":0,"query":{"perPage":10,"pages":0,"offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-query alignwide"><!-- wp:query-no-results -->
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center">No search results.</p>
```

Comment:
We should make this into a pattern to reuse across templates, and wrap this text in translation function.